### PR TITLE
Feature/documentation

### PR DESCRIPTION
**Pull request by joakim-olesen**

### CODE REVIEW

**Comment by nightkr** [comment]
Yeah, sadly GitHub doesn't support tables of contents, so setting the flag (which is on by default, anyway) won't be of much help. As a workaround you can bake a table-of-contents into the file (and [toc-org](https://github.com/snosov1/toc-org) will keep it up-to-date for you), but that's another Emacs dependency when editing the file.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 /target
 **/*.rs.bk
+bin
```
Comment (nightkr) [inline .gitignore]: I'm curious about this.. does Visual Studio not build with Cargo? Normally all Rust tooling uses Cargo, which places all build artifacts inside `target`.

Comment (joakim-olesen) [reply .gitignore]: That rule is not necessary actually. I pushed a new commit where it's deleted.

**Comment by joakim-olesen** [comment]
> Yeah, sadly GitHub doesn't support tables of contents, so setting the flag (which is on by default, anyway) won't be of much help. As a workaround you can bake a table-of-contents into the file (and [toc-org](https://github.com/snosov1/toc-org) will keep it up-to-date for you), but that's another Emacs dependency when editing the file.

I see, thanks for the info! (I won't spend time on that now though.)

**Pull request merged**